src/pages/base-page.ts

```diff
@@ -49,6 +49,11 @@ export class BasePage {
         return this.current.byElementText(tagName, text);
     }
 
+    public bySystelabTestId(systelabTestId: string): ElementFinder {
```

Comment:
Hi Fran, thanks for your PR! Looks great, but I have just a question. Do we want to keep the systelab name on the method name? Or could we change it for?
byDataTestId
allByDataTestId

Just to follow the other method names that we have on the base-page.ts nad widget.ts (byId, by TagName...)
![image](https://user-images.githubusercontent.com/48098715/235162177-2c2c0a95-302a-41f8-966d-e6be04ea5e89.png)

What do you think? Again, thanks a lot. That could be great for all the teams :-)

Reply:
Names proposed in issue's description were used